plan/task/exec.go

```diff
@@ -150,19 +153,46 @@ func (t *stopAsyncExecTask) Run(ctx context.Context, pctx *plancontext.Context,
 		return nil, err
 	}
 
+	timeout, err := v.Lookup("timeout").Int64()
+	if err != nil {
+		return nil, err
+	}
+
 	lg := log.Ctx(ctx)
 
-	exitCode, err := s.StopContainer(ctx, ctrID)
+	exitCode, err := s.StopContainer(ctx, ctrID, time.Duration(timeout))
 	if err != nil {
-		return nil, fmt.Errorf("failed to stop async exec %s: %w", ctrID, err)
+		return nil, fmt.Errorf("failed to stop exec %s: %w", ctrID, err)
```

Comment:
It is still an asynchronous execution, why did you remove that from log & error??

Reply:
Changed it because even though it is asynchronous, that's not the terminology we ended up using, so I thought that using `async` in the user-facing error message might be slightly confusing.